index.html

```diff
@@ -0,0 +1,216 @@
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>What is Microverse? - YouTube</title>
+
+    <link rel="stylesheet" href="style.css" />
+    <link
+      rel="stylesheet"
+      href="https://cdn.jsdelivr.net/npm/bootstrap@4.0.0/dist/css/bootstrap.min.css"
+      integrity="sha384-Gn5384xqQ1aoWXA+058RXPxPg6fy4IWvTNh0E263XmFcJlSAwiGgFAW/dAiS6JXm"
+      crossorigin="anonymous"
+    />
+    <link
+      href="https://unpkg.com/boxicons@2.1.2/css/boxicons.min.css"
+      rel="stylesheet"
+    />
+  </head>
+  <body>
+    <nav id="header">
+      <section class="left">
+        <i class="bx bx-menu" id="btn" style="font-size: 30px"></i>
+        <img
+          src="images/youtube.png"
+          alt=""
+          style="width: 100px; padding-bottom: 20px"
+        />
+      </section>
+      <section class="middle">
+        <input type="text" id="search" placeholder="Search " />
+        <i
+          class="bx bx-search"
+          id="search-btn"
+          style="
+            width: 50px;
+            text-align: center;
+            font-size: 20px;
+            color: gray;
+            font-size: 38px;
+            border: 1px solid #ccc9c9;
+            align-items: center;
+          "
+        ></i>
+        <i
+          class="bx bxs-microphone"
+          style="font-size: 20px; padding-left: 10px; padding-top: 10px"
+        ></i>
+      </section>
+      <section class="right">
+        <span class="right-hand-nav-items" >
+          <!-- create video -->
+          <a href="#"><i class="bx bx-video-plus" alt="Notifications"></i></a>
+          <!-- Notification -->
+          <a href="#"></a><i class="bx bx-bell"></i></a>
+          <!-- My account -->
+          <a href="#">
+            <img
+                    src="images/account-pic.jpg"
+                    alt="Holder's account photo."
+                    style="border-radius: 100%; width: 40px; padding-bottom: 15px;"
+                  /></a>
+        </span>
+      </section>
+    </nav>
+
+    <div class="container">
+      <section class="main-content">
+        <article id="main-article">
+          <section style="text-align: center; width: 100%">
+            <iframe
+              width="640"
+              height="360"
+              src="https://www.youtube.com/embed/f9ip1iSAptQ"
+              title="YouTube video player"
+              frameborder="0"
+              allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture"
+              allowfullscreen
+            ></iframe>
+          </section>
+          <h1 style="padding-top: 10px" ;>What is Microverse?</h1>
+          <section id="video-caption">
+            <b>123,788 views Mar 12, 2019</b> Study online for a period of time
+            between 8-10 months without paying a cent. Only pay when you land
+            yourself a job that pays more than<b>...more</b>
+          </section>
```

Comment:
- Please remove the article as it's not applicable to your content. An article in a webpage makes sense on it's own because it describes content that is independent of the web page and your webpage doesn't seem to have that content within itself .
- Please removed the nested sections. This is because you can't have a section without an h1 title tag. Based on your content, you may only one section.